rag/flow/chunker/token_chunker.py

```
@@ -230,19 +230,59 @@ def _attach_context_to_media_chunks(chunks, table_context_size, image_context_si
+    spans = []
+    offset = 0
+    for item_text, _pos_group in prev_items:
+        start = offset
+        end = offset + len(item_text)
+        spans.append((start, end))
+        offset = end + 1
+    if not spans:
+        return []
+    total = spans[-1][1]
+    out = []
+    for (start, end), (_text, pos_group) in zip(spans, prev_items):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)